README.md

```diff
@@ -94,10 +72,15 @@ You can call `ex.run_with_retry(opts)` on an individual example.
 - __:exceptions_to_hard_fail__(default: *[]*) List of exceptions that will trigger an immediate test failure without retry. Takes precedence over __:exceptions_to_retry__
 - __:exceptions_to_retry__(default: *[]*) List of exceptions that will trigger a retry (when empty, all exceptions will)
 - __:retry_callback__(default: *nil*) Callback function to be called between retries
+- __:retry_on_failure__(default: *main? || pr?*) Retry examples on failure. This is useful for flaky tests that are not marked with `:retry` metadata.
```

Comment:
Probably worth mentioning in the README which ENV vars act as controls for `main?` and `pr?`